server/grpc/api/mapping.go

```diff
@@ -175,7 +182,7 @@ func toProtoParameters(params []*service.Parameter) []*definition.Parameter {
 
 func toProtoConfiguration(configuration *service.Dependency) *definition.Configuration {
 	if configuration == nil {
```

Comment:
same here I guess, it shouldn't' check for nil, as this function is for mapping, not nic catching :)

Reply:
Same answer, this have to be managed somewhere but i agree that it's not the best place here.